src/js/shared/reducers/shot-generator.js

```diff
@@ -691,6 +695,26 @@ module.exports = {
             draft.sceneObjects[action.payload.id].distance = action.payload.distance
           }
 
+          if (action.payload.numberOfLayers != null) {
+            draft.sceneObjects[action.payload.id].numberOfLayers = action.payload.numberOfLayers
+          }
+
+          if (action.payload.distanceBetweenLayers != null) {
+            draft.sceneObjects[action.payload.id].distanceBetweenLayers = action.payload.distanceBetweenLayers
+          }
+
+          if (action.payload.effect != null) {
```

Comment:
What is `effect`? Should it be renamed `volumePresetId`?

Reply:
changed, was left from when i started adding effects and they just had names